### ISSUE TITLE
Consider adding pgTAP tests

### ISSUE THREAD

**Issue by steve-chavez** [point-source/supabase-tenant-rbac#1]
Some [pgTAP](https://pgtap.org/) tests would be great here to better understand the functionality.

References:

- Guide: https://supabase.com/docs/guides/database/extensions/pgtap
- Other examples: https://github.com/steve-chavez/socnet/tree/master/tests

**Comment by point-source** [comment]
Agreed. I'd be happy to accept a PR with this functionality. Or perhaps one of the many ai tools out there can assist with generating some of this. Otherwise, this will have to happen as time allows and that's probably a ways off :(

**Comment by point-source** [comment]
Tests have been added
